### PR TITLE
fix: alert box added when the user wants to delete card

### LINKED ISSUE #613: fix: no alert box when user chooses to delete card
**Summary:**
When the user wishes to delete card, he should be prompted if he wants to confirm about deleting the card by an alert box.

**Steps to reproduce:**

1. Go to cards
2. Try to delete a card
3. It deleted automatically

### PR DESCRIPTION
**Pull request by Bhavnaharitsa**
Fixes #613 

ezgif com-video-to-gif

- [/] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [/] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [/] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by Bhavnaharitsa** [comment]
@luckyman20 Please help me with the Travis errors

**Comment by luckyman20** [comment]
You can checkout the errors [here](https://travis-ci.com/openMF/mobile-wallet/builds/142008829) (from line 2019)

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/Constants.java [hunk 1]
```diff
@@ -182,6 +184,7 @@
     public static final String ERROR_FIELDS_CANNOT_BE_EMPTY = "Fields cannot be empty";
     public static final String ERROR_VALIDATING_PASSWORD = "Passwords are not the same";
 
+
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/Constants.java]:
```suggestion
```

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/Constants.java [hunk 2]
```diff
@@ -136,6 +136,8 @@
     public static final String GOOGLE_PHOTO_URI = "GOOGLE_PHOTO_URI";
     public static final String CHOOSE_SIGNUP_METHOD = "Choose Signup Method";
     public static final String LOGGING_OUT = "Logging out...";
+    public static final String DELETE_CARD = "Deleting card...";
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/Constants.java]:
```suggestion
    public static final String DELETE_CARD = "Deleting the card...";
```

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/Constants.java [hunk 3]
```diff
@@ -136,6 +136,8 @@
...
+
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/Constants.java]:
```suggestion
```

File: mifospay/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -233,6 +233,7 @@
     <string name="password_strong">Strong</string>
     <string name="password_very_strong">Very Strong</string>
     <string name="amount">Amount</string>
+    <string name="alert_delete_card">Do you want to delete card?</string>
```
Comment (luckyman20) [inline mifospay/src/main/res/values/strings.xml]:
```suggestion
    <string name="alert_delete_card">Are you sure you want to delete the card?</string>
```

**Comment by Bhavnaharitsa** [comment]
Sure closing this for now 
Opening a new pr synced with the latest changes

**Pull request closed without merging**